inc/Engine/Media/AboveTheFold/AJAX/Controller.php

```diff
@@ -284,13 +284,32 @@ private function validate_image_src( $image_src ) {
 			return false;
 		}
 
+		if ( preg_match( '/^chrome-[^:]+:\/\/[a-zA-Z0-9\-_]+\/?/', $image_src ) ) {
+			return false;
+		}
+
 		// Here we get the url PATH part only to strip all query strings.
 		$image_src_path = wp_parse_url( $image_src, PHP_URL_PATH );
 		if ( empty( $image_src_path ) ) {
 			return false;
 		}
+		
+		// Add svg to allowed mime types.
+		add_filter( 'mime_types', [ $this, 'add_svg_to_mime' ] );
```

Comment:
This will add the support of svg in the whole site not that specific feature, do we really want to do that or we just need it to be in our feature only?

Reply:
Just for our feature. I agree we should not touch the mime_types filter globally